scripts/fuzz_opt.py

```diff
@@ -138,7 +139,16 @@ def randomize_fuzz_settings():
         FUZZ_OPTS += ['--legalize-js-interface']
     else:
         LEGALIZE = False
-    print('randomized settings (NaNs, OOB, legalize):', NANS, OOB, LEGALIZE)
+    extra_v8_opts = []
+    if random.random() < 0.5:
+        if random.random() < 0.5:
+            # test the optimizing compiler
+            extra_v8_opts += ['--no-liftoff']
+        else:
+            # test the baseline compiler
+            extra_v8_opts += ['--liftoff', '--no-wasm-tier-up']
```

Comment:
So what if we don't give any options (with 50% probability)? We run the interpreter? So now the probability is
- 0.5: interpreter
- 0.25: baseline compiler
- 0.25: optimizing compiler
?

If so, do we need to give 0.5 probability to the interpreter? Maybe 1/3 each? Anyway, it'd be better to state these probabilities in comments.

Reply:
good idea, I added detailed comments now, let me know what you think.

Reply:
Thanks, this is easier to follow!